src/screens/SelectMultiple.js

```diff
@@ -254,6 +255,11 @@ const SelectMultiplePage = () => (
 
         <Property name="messages">
           <Description>Custom messages.</Description>
+          <Paragraph>
+            To affect all SelectMultiple instances, use{' '}
+            <RoutedAnchor path="/grommet#messages" label="messages prop" />, on
```

Comment:
```suggestion
            <RoutedAnchor path="/grommet#messages" label="messages prop" />{' '}on
```
removing the comma. don't think it should be there gramatically.